docs/explanation/lifecycle.rst

```diff
@@ -2,7 +2,62 @@
 Lifecycle details
 *****************
 
+Each part is built in five separate steps:
+
+#. ``PULL`` — The components for the part are retrieved from their stated
+   location and placed into a package cache area.
+#. ``OVERLAY`` — Any overlay packages are incorporated into the staging area
```

Comment:
What is the `staging area` referenced here?

Reply:
This is supposed to mean the overlay directory. I've rephrased it, please lmk what you think now.